### PR TITLE
[Bot] Update tasks specs

### PR DESCRIPTION
**Pull request by HuggingFaceInfra**
This PR updates the @huggingface/tasks specs. It has been generated by running:
```sh
pnpm run inference-tgi-import
pnpm run inference-tei-import
pnpm run inference-codegen
```

This PR was automatically created by the [Tasks - Update specs workflow](https://github.com/huggingface/huggingface.js/blob/main/.github/update-specs.yml).

Make sure the changes are correct before merging.

### CODE REVIEW

**Review by coyotte508 — APPROVED** [review]

**Comment by Wauplin** [comment]
This PR has been created as part of https://github.com/huggingface/huggingface.js/pull/964. Let's not merge for now.

File: packages/tasks/src/tasks/chat-completion/inference.ts [hunk 1]
```diff
@@ -18,6 +18,10 @@ export interface ChatCompletionInput {
 	 * decreasing the model's likelihood to repeat the same line verbatim.
 	 */
 	frequency_penalty?: number;
+	/**
+	 * A guideline to be used in the chat_template
+	 */
+	guideline?: string;
```
Comment (Wauplin) [inline packages/tasks/src/tasks/chat-completion/inference.ts:24]: Not a fan of this argument as it's not used anymore. Best solution would be to remove it from TGI specs (see [slack thread](https://huggingface.slack.com/archives/C05CFK1HM0T/p1732009523291719?thread_ts=1724852501.099809&cid=C05CFK1HM0T)). Let's wait without merging this PR for now.

Comment (Wauplin) [reply packages/tasks/src/tasks/chat-completion/inference.ts:24]: opened https://github.com/huggingface/text-generation-inference/pull/2762 to remove `guideline` from TGI API

**Review by Wauplin — APPROVED** [review]
Looks good now :)

**Pull request merged**